internal/provider/common_test.go

```diff
@@ -22,8 +23,8 @@ func checkAPIServerConnection(t *testing.T) {
 // to create a provider server to which the CLI can reattach.
 func providerFactories() map[string]func() (*schema.Provider, error) {
 	return map[string]func() (*schema.Provider, error){
-		"kentik-cloudexport": func() (*schema.Provider, error) {
-			return New(), nil
+		"kentik-cloudexport": func() (*schema.Provider, error) { //nolint: unparam
```

Comment:
does underscore for unused param silence the linter as well?

Reply:
The warning is following:
```
internal/provider/common_test.go:26:51: providerFactories$1 - result 1 (error) is always nil (unparam)
"kentik-cloudexport": func() (*schema.Provider, error) {
```

Using underscore return values does not help for that